server/liveblog/themes/themes_assets/default/gulpfile.js

```diff
@@ -288,6 +288,15 @@ const lessCommon = (cleanCss) => {
   const lessFiles = [];
   let themeLess;
 
+  // check if there is _topImports.less file with font @imports etc.
+  // that should be put on very top of compiled css to work
+  if ( !theme.onlyOwnCss && theme.extends ) {
+    themeLess = path.resolve(`./less/_topImports.less`);
```

Comment:
can you please change to a better name? like `topImport`

Reply:
ok, uno momentu